gapic/templates/tests/unit/%name_%version/%sub/test_%service.py.j2

```diff
@@ -624,8 +621,13 @@ def test_{{ service.name|snake_case }}_grpc_transport_channel_mtls_with_client_c
     assert transport.grpc_channel == mock_grpc_channel
 
 
+@pytest.mark.parametrize(
+    "api_mtls_endpoint", ["mtls.squid.clam.whelk", "mtls.squid.clam.whelk:443"]
+)
```

Comment:
I didn't know you could do this. Neat!

Reply:
I am glad that pytest has parametrized test too :)